x/wasm/keeper/api.go

```diff
@@ -30,14 +28,14 @@ func (a cosmwasmAPIImpl) humanAddress(canon []byte) (string, uint64, error) {
 	gas := a.gasMultiplier.FromWasmVMGas(5)
 	if err := sdk.VerifyAddressFormat(canon); err != nil {
 		//nolint:stylecheck
```

Comment:
This is very small thing. Does this `nolint` need?

Reply:
It looks good to delete it.